### PR TITLE
Updated PrestaShop documentation

### PR DESCRIPTION
**Pull request by BitcoinMitchell**

### CODE REVIEW

**Comment by BitcoinMitchell** [comment]
Current broken links are unrelated to this PR.

File: docs/PrestaShop.md [hunk 1]
```diff
@@ -5,49 +5,56 @@ If you do not have a store yet, follow [this step by step article](https://blog.
 
 To integrate BTCPay Server into an existing PrestaShop store, follow the steps below.
 
-## 0. Server Requirements
+**PSA:** This document only applies to the latest _v5_ version of the module. If you want the _v4_ module documentation, [click here](https://github.com/btcpayserver/btcpayserver-doc/blob/cba96292ceea9483711ab53c479a98357383f857/docs/PrestaShop.md).
```
Comment (pavlenex) [inline docs/PrestaShop.md:8]:
```suggestion
:::tip
This document only applies to the latest _v5_ version of the module. If you want the _v4_ module documentation, [click here](https://github.com/btcpayserver/btcpayserver-doc/blob/cba96292ceea9483711ab53c479a98357383f857/docs/PrestaShop.md).
:::
```

**Review by pavlenex — APPROVED** [review]

**Comment by pavlenex** [comment]
LGTM, what would be really cool is that we automatically pull this doc from the presta readme in the future. cc @dennisreimann

**Comment by dennisreimann** [comment]
We can move it there and pull it it. Depends on what's most convenient for the maintainer. As the repo does not belong to us, we'd have trouble updating the docs, whenever the PrestaShop docs in another repo change. We'd have to manually trigger the docs update on our side.

**Comment by pavlenex** [comment]
I wasn't even aware that the repo doesn't belong to btcpay organizaiton. @BitcoinMitchell For visibility  and ease of maintaining if you'd like we can move it and make you owner of it, but this isn't something that we have to do now.

**Comment by BitcoinMitchell** [comment]
> I wasn't even aware that the repo doesn't belong to btcpay organizaiton. @BitcoinMitchell For visibility and ease of maintaining if you'd like we can move it and make you owner of it, but this isn't something that we have to do now.

I don't follow. The prestashop-plugin exists in: https://github.com/btcpayserver/prestashop-plugin

File: docs/PrestaShop.md (hunk 1 above)
Comment (BitcoinMitchell) [reply docs/PrestaShop.md]: Thank you, great change!

**Comment by dennisreimann** [comment]
Ah ok, sorry my bad then!

**Pull request merged**